x/process/internal/keeper/keeper.go

```diff
@@ -72,13 +68,7 @@ func (k Keeper) Create(ctx sdk.Context, msg *types.MsgCreate) (*processpb.Proces
 		}
 	}
 
-	procInitBal, err := sdk.ParseCoins(processCreateInitialBalance)
-	if err != nil {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidCoins, err.Error())
-	}
-	if err := k.bankKeeper.SendCoins(ctx, msg.Owner, p.Address, procInitBal); err != nil {
-		return nil, err
-	}
+	p.PaymentAddress = msg.Owner
```

Comment:
this should be set in the function `process.New`.
Once a resource is created, we should avoid as much as possible to modify it.